pyproject.toml

```
@@ -8,14 +8,15 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "3.6"
+python = "^3.6"
```

```suggestion
python = "~3.8.0"
```
Let a stick to single version of python. If we do `^3.6` and if system has python3.10 then poetry picks up 3.10 version.